catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/pom.xml

```diff
@@ -101,6 +94,7 @@
             <artifactId>spatial-wfs-v1_1_0-common</artifactId>
             <version>${project.version}</version>
             <exclusions>
+                <!-- FIXME: There's no cxf-api artifact anymore -->
```

Comment:
✏️ Do these FIXMEs need to be addressed?

Reply:
I removed the exclusions. Since the artifact doesn't exist the exclusions didn't do anything and it still worked